build.gradle

```
@@ -1,5 +1,5 @@
 plugins {
-    id 'fabric-loom' version '0.9.45'
+    id 'fabric-loom' version '0.10-SNAPSHOT'
```

```suggestion
    id 'fabric-loom' version '0.10.+'
```

Please consider `-SNAPSHOT` as deprecated.
Also, I'm not sure you actually tested this buildscript since this version of loom-quiltflower have some loom version constraints.
It may be switched for mini loom quiltflower though.